tools/istio-iptables/pkg/builder/iptables_builder.go

```diff
@@ -40,6 +40,10 @@ type IptablesConsumer interface {
 
 // IptablesBuilder is a higher level interface based on builder pattern.
 type IptablesBuilder interface {
+	// NewChainV4 creates new IPv4 chain
+	NewChainV4(chain string, table string) IptablesBuilder
+	// NewChainV6 creates new IPv6 chain
+	NewChainV6(chain string, table string) IptablesBuilder
 	IptablesProducer
```

Comment:
We usually list inherited types first, before method declaration. Just move `IptablesProducer` and `IptablesConsumer` above the method declarations.

Reply:
Fixed order